plugin.go

```diff
@@ -103,9 +102,14 @@ func installCaCert(cacert string) *exec.Cmd {
 	)
 }
 
-func exportSecrets(secrets map[string]string) {
-	for k, v := range secrets {
-		os.Setenv(fmt.Sprintf("%s", k), fmt.Sprintf("%s", os.Getenv(v)))
+func CopyTfEnv() {
+	tfVar := regexp.MustCompile(`^TF_VAR_.*$`)
+	for _, e := range os.Environ() {
+		pair := strings.Split(e, "=")
+		if tfVar.MatchString(pair[0]) {
+			name := strings.Split(pair[0], "TF_VAR_")
+			os.Setenv(fmt.Sprintf("TF_VAR_%s", strings.ToLower(name[1])), pair[1])
```

Comment:
I agree.  And I meant to bring it up to @bradrydzewski (hi brad) ... this was going to be more of a "band-aid" in the meantime.